docs/user-guide/quickstart.md

```diff
@@ -0,0 +1,157 @@
+# SINCLAIR Quickstart
+
+## Overview
+
+**SINCLAIR** is an end-to-end NextFlow pipeline for processing single cell RNASeq (scRNASeq) data from either raw fastq.gz files or count files in `.h5` format, as produced by the 10X CellRanger pipeline, and primarily uses [Seurat v5](https://www.satijalab.org/Seurat) as its backbone for downstream processing.
+
+In short, SINCLAIR performs the following functions:
+
+- Alignment from FASTQ (optional)
+- Initial quality control and cell filtering per sample
+- Sample combination
+- Batch correction
+- Preliminary cell type annotation
+- Preliminary clustering
+
+The final outputs are a set of Seurat `.rds` files that contain all provided samples with and without batch correction, with the latter evaluated with several algorithms.
+
+## Installation and Initialization
+
+### Via CCBRPipeliner on Biowulf (NIH HPC)
+
+If working on Biowulf, start an interactive session with a minimum of 16 CPUs, 8 hours wall-time, and a local scratch allocation (temporary RAM) of 128 GB:
+
+```
+sinteractive --mem=64g --cpus-per-task=16 --time=8:00:00 --gres=lscratch:128
+```
+
+As of CCBR Pipeliner v8, instantiate Pipeliner as a module:
+
+```
+module load ccbrpipeliner
+```
+
+Navigate to your working directory and initialize SINCLAIR:
+
+```
+sinclair init
+```
+
+### From GitHub
+
+Clone the repository from GitHub:
+
+```
+git clone https://github.com/CCBR/SINCLAIR
+```
+
+Additional required dependencies:
+
+- NextFlow
+- Singularity
+
```

Comment:
Let's remove this from the quickstart and put it in a separate doc for advanced users. The vast majority won't need this and may get confused by having it here.

```suggestion
```